approval_polls/templates/approval_polls/poll_update_form.html

```diff
@@ -0,0 +1,69 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div class='row-fluid top-buffer'>
+        <div class='col-md-12'>
+            <h3> Edit Poll </h3>
+        </div>
+    </div>
+    <div class='row-fluid top-buffer'>
+        <div class='col-md-12'>
+            <form action="" method="post"> {% csrf_token %}
+                {{ form.non_field_errors }}
+                <div class="fieldWrapper">
+                    <label>Poll Active/Inactive:</label>
+                    {{ form.suspended }}
+                    {% if form.suspended.errors %}
+                        <div class="alert alert-danger" role="alert">
+                            {{ form.suspended.errors }}
+                        </div>
+                    {% endif %}
+                </div>
+                <div class="fieldWrapper">
+                    <label for="{{ form.question.id_for_label }}">Question: </label>{{ form.question }}
+                    {% if form.question.errors %}
+                        <div class="alert alert-danger" role="alert">
+                            {{ form.questions.errors }}
+                        </div>
+                    {% endif %}
+                </div>
+                <div class="fieldWrapper">
+
+                    <label for="{{ form.open_date.id_for_label }}">Open Poll: </label>{{ form.open_date }}
+                    {% if form.open_date.errors %}
+                        <div class="alert alert-danger" role="alert">
+                            {{ form.open_date.errors }}
+                        </div>
+                    {% endif %}
+                </div>
+                <div class="fieldWrapper">
+                    <label for="{{ form.close_date.id_for_label }}">Close Poll: </label>{{ form.close_date }}
+                    {% if form.close_date.errors %}
+                        <div class="alert alert-danger" role="alert">
+                            {{ form.close_date.errors }}
+                        </div>
+                    {% endif %}
+
+                </div>
+
+                {% for choice in choices %}
+                    <label for="{{ choice.form.choice_text.id_for_label }}.{{ choice.id }}">Choice {{ forloop.counter }}: </label>
+                    <input class="form-control" id="id_choice_text.{{ choice.id }}" name="choice_text.{{ choice.id }}"
+                           type="text" value="{{ choice.form.choice_text.value }}"/>
+                    {% if choice.choice_text.errors %}
+                        <div class="alert alert-danger" role="alert">
+                            {{ choice.choice_text.errors }}
+                        </div>
+                    {% endif %}
+                {% endfor %}
+                <input type="submit" class="btn btn-default" value="Update"/>
```

Comment:
Yes I can use a button instead of an input. It's just an old school habit using and input. :)